src/CLIOptions.cpp

```diff
@@ -145,6 +145,11 @@ void setCLIOptions(CLI::App &App, MainOpt &MainOptions) {
   addUriOption(App, "--graylog-logger-address",
                MainOptions.GraylogLoggerAddress,
                "<host:port> Log to Graylog via graylog_logger library");
+  addUriOption(App, "--grafana-carbon-address", MainOptions.GrafanCarbonAddress,
+               "<host:port> Address to the Grafana (Carbon) metrics service.");
+  App.add_option("--metrics-id", MainOptions.GrafanaCarbonMetricsId,
+                 "An extra metrics ID string. Only used if present. Will take "
+                 "the form: \"kafka-to-nexus.[id].*\"");
```

Comment:
Tobias has requested reintroduction of a configurable "service_id" in status messages. Perhaps we should add that as an option instead (and always use it as part of the metrics name if it is set)?

Reply:
Sure.